test/msw-api/setup-worker/life-cycle-events/on.mocks.ts

```diff
@@ -22,6 +25,12 @@ worker.events.on('request:unhandled', (req) => {
   console.warn(`[request:unhandled] ${req.method} ${req.url.href} ${req.id}`)
 })
 
+worker.events.on('request:unhandledException', (req, handler, error) => {
+  console.warn(
+    `[request:unhandledException] ${req.method} ${req.url.href} ${req.id} ${handler.info.header} ${error.message}`,
```

Comment:
`handler.info.header` is synonymous to `req.method req.url.href` at the beginning of this string. I think we can drop `header.info.header` from this warning.

Reply:
It is true that it makes no sense to add `handler.info.header` to the warning in a real scenario.

However I want to test that `handler` was actually passed and its properties can be used. `handler.info.header` makes the most sense to me as it is the only property that has a value related to the test setup (e.g. uses correct url). `handler.markAsSkipped` has no meaning here.

Reply:
Since we will remove handler from the arguments of the event, this problem does not exist anymore